.neutrinorc.js

```diff
@@ -4,18 +4,18 @@ module.exports = {
       hot: false,
       babel: {
         plugins: [
-          require.resolve('babel-plugin-transform-object-rest-spread'),
+          require.resolve('@babel/plugin-proposal-object-rest-spread'),
```

Comment:
`plugin-proposal-object-rest-spread` is now included in `@babel/preset-env`, so should work out of the box :-)